sriovnet.go

```diff
@@ -2,13 +2,14 @@ package sriovnet
 
 import (
 	"fmt"
-	"github.com/satori/go.uuid"
-	"github.com/vishvananda/netlink"
 	"log"
 	"net"
 	"path/filepath"
 	"strconv"
 	"strings"
+
```

Comment:
@moshe010 this change is not related.

go fmt with go1.12.4 doesnt complain about making above change.
Any reason to move them?